provisioning/tasks/install-site.yml

```diff
@@ -14,7 +14,7 @@
     --site-name="{{ drupal_site_name }}"
     --account-name={{ drupal_account_name }}
     --account-pass={{ drupal_account_pass }}
-    --db-url=mysql://{{ drupal_mysql_user }}:{{ drupal_mysql_password }}@localhost/{{ mysql_databases[0].name }}
+    --db-url={{ drupalvm_database }}://{{ drupal_mysql_user }}:{{ drupal_mysql_password }}@localhost/{{ drupal_mysql_database }}
```

Comment:
`mysql_databases[0]` references the `drupal_mysql_database` so feels we should use that variable instead. Currently this is a bug where drush installs it into the first database defined regardless of what the `drupal_mysql_database` is set to.

Yes renaming these variables would be better imo, but if we just want to add support for PostgreSQL with minimal changes we can keep using these.